Student/StudentE2ETests/Planner/PlannerTests.swift

```diff
@@ -56,7 +56,7 @@ class PlannerTests: CoreUITestCase {
 
         PlannerList.event(id: "2233").tap()
         app.find(label: "first").waitToExist()
-        XCTAssert(app.find(label: "Instructure SLC").exists())
+        app.find(label: "Instructure SLC").waitToExist()
```

Comment:
Could you make this change in the Parent version of PlannerTests too?

Reply:
Oh right, thanks!